db/migrate/20200120120940_add_course_from_find_id_to_candidate_model.rb

```diff
@@ -0,0 +1,9 @@
+class AddCourseFromFindIdToCandidateModel < ActiveRecord::Migration[6.0]
+  def up
+    add_column :candidates, :course_from_find_id, :integer, default: nil
```

Comment:
You could use https://apidock.com/rails/v6.0.0/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference here, which is reversible / adds some database constraints.

```rb
add_reference(:candidates, :course_from_find, foreign_key: { to_table: :courses }, foreign_key: true)
```